### PR TITLE
Have another swing at new event logging.

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

**Review by mandoonandy — APPROVED** [review]
Only a quick look but LGTM. One spelling error noticed because I had to say something.

**Comment by mikalstill** [comment]
I'm not sure why CI is so flakey. It feels like a bug in triggers that I've accidentally made worse to me.

**Comment by mikalstill** [comment]
Then I remembered that triggers change is in a different PR. I'm still not sure why the CI is flakey, I continue to chase.

File: shakenfist/etcd.py [hunk 1]
```diff
@@ -44,6 +44,23 @@ def __init__(self, host='localhost', port=2379, protocol='http',
             cert_key=cert_key, cert_cert=cert_cert, timeout=timeout,
             api_path=api_path)
 
+    # Replace the upstream implementation with one which allows for limts on range
```
[the remark this answers is not in the document]
Comment (mikalstill) [reply shakenfist/etcd.py:47]: Done.

File: shakenfist/etcd.py [hunk 2]
```diff
@@ -203,19 +220,12 @@ def __enter__(self):
             if res:
                 duration = time.time() - start_time
                 if duration > threshold:
-                    db.add_event(self.objecttype, self.objectname,
```
[the remark this answers is not in the document]
Comment (mikalstill) [reply shakenfist/etcd.py:206]: Well, there is a log entry just below...

File: shakenfist/etcd.py [hunk 3]
```diff
@@ -228,9 +238,6 @@ def __enter__(self):
             time.sleep(1)
 
         duration = time.time() - start_time
-        db.add_event(self.objecttype, self.objectname,
```
[the remark this answers is not in the document]
Comment (mikalstill) [reply shakenfist/etcd.py:231]: There is a log entry for this one too in a few more lines...

**Comment by mikalstill** [comment]
I think the CI problems here were actually the CI cluster underneath. Here's hoping at least.

**Pull request merged**